rpmlb/builder/koji.py

```diff
@@ -43,14 +50,23 @@ def __init__(
         self.epel = koji_epel
 
         #: Format string for the build target
-        self.target_format = koji_target_format
+        if koji_target_format is None:
+            self.target_format = self.DEFAULT_TARGET_FORMAT
+        else:
+            self.target_format = koji_target_format
```

Comment:
Theoretically yes, but I doubt we will ever encounter configuration profile named `<empty string>`… so I will go with the first option.

The "ternary operator" was my first choice, but the line length was through the roof, so I have rewritten it to the `if` you see now.